### ISSUE TITLE
Cannot read property 'setSubscriptionKeys' of undefined

### ISSUE THREAD

**Issue by Chos89** [msavin/JetSetter#6]
Getting this error on my console, using windows meteor.

Uncaught TypeError: Cannot read property 'setSubscriptionKeys' of undefined
ToyKit.js:59 
MeteorToys_ToyGlue.runMongol
ToyKit.js:4 
MeteorToys_ToyGlue.observe
ToyKit.js:65 
(anonymous function)
startup_client.js:30 
runStartupCallbacksstartup_client.js:32 ready

List of the packages i'm using:

accounts-password             1.1.0-win.1\* Password support for accounts
dfischer:prerenderio          1.0.5  Node wrapper for prerenderio
fortawesome:fontawesome       4.2.0_2  Font Awesome (official): 470+ scalable vector icons, customizable via CSS, Retina friendly
iron:router                   1.0.7  Routing specifically designed for Meteor
jquery                        1.11.3-rc.0_2\* Manipulate the DOM using CSS selectors
kevohagan:sweetalert          0.4.2  a beautiful replacement for javascript's alert()
matteodem:easy-search         1.4.13  Easy-to-use search with Blaze Components (+ Elastic Search support)
meteor-platform               1.2.2-rc.0\* Include a standard set of Meteor packages in your app
meteorhacks:search-source     1.2.0  Reactive Data Source for Search
momentjs:moment               2.9.0  Moment.js (official): parse, validate, manipulate, and display dates - official Meteor packaging
msavin:jetsetter              1.0.0  Visual Get/Set Tool for Sessions
msavin:mongol                 0.7.2\* The insanely handy development package for Meteor.
stylus                        1.0.7-rc.0\* Expressive, dynamic, robust CSS
summernote:summernote         0.6.0_1  summernote (official): jQuery+Bootstrap+FontAwesome WYSIWYG editor with embedded images support
twbs:bootstrap                3.3.2\* Bootstrap (official): the most popular HTML/CSS/JS framework for responsive, mobile first projects
vazco:universe-html-purifier

**Comment by hyperborea** [comment]
Same here on normal (non-windows) Meteor.

**Comment by hyperborea** [comment]
The issue here is using an outdated mongol package. Before commit 1e19032 mongol wasn't exporting the MongolPackage object, at least since version 1.0.7 it does and works perfectly.

Strangely enough meteor update didn't do the trick for me, I needed to remove and re-add mongol to get the correct version in place.

**Comment by Chos89** [comment]
Confirmed, removing and adding back mongol did the trick

**Comment by msavin** [comment]
Great news!

**Comment by mizzao** [comment]
I think the issue is that jetsetter doesn't work without mongol. There is some dependency from either jetsetter or meteor-toys that tries to access mongol without checking if it is added to the app first.

**Comment by msavin** [comment]
@mizzao they both share the same kernel (ToyKit). There seem to be some issues when someone tries to install one package that has a newer kernel version than the other. Removing & re-adding seems to work it, whereas (from what I've heard) an  update does not. I did test both JS + M individually just now and they seem to work. Are you having any issues with either?

**Comment by mizzao** [comment]
@msavin I added jetsetter to a project that had no prior packages of yours, and saw that it installed ToyKit. However, starting the app showed an error in the console that involved `Packages["msavin:mongol"]`. So I added mongol too (which installed a bunch of other stuff) and the error went away, and both work.

Not sure if it is a dependency issue or just a one-off weirdness that I observed, possibly from meteor pulling in a bad version.
